tests/unittest/_torch/speculative/test_eagle3.py

```
@@ -92,39 +92,72 @@ def test_kv_lens_runtime_with_eagle3_one_model():
+        [True, "TRTLLM", True, False, True, False, False, False, False, True],
+        [False, "TRTLLM", False, True, True, True, False, True, False, True],
+        [False, "TRTLLM", True, False, True, False, False, True, False, True],
     ])
 @pytest.mark.high_cuda_memory
 def test_llama_eagle3(use_cuda_graph: bool, attn_backend: str,
                       disable_overlap_scheduler: bool, enable_block_reuse: bool,
                       use_one_model: bool, enable_chunked_prefill: bool,
                       use_chain_drafter: bool, multi_batch: bool,
-                      attention_dp: bool, request):
+                      attention_dp: bool, use_separate_draft_kv_cache: bool,
+                      request):
```

Ruff (0.14.10)

Unused function argument: `request`

(ARG001)